dali/c_api/c_api.cc

```diff
@@ -130,6 +130,27 @@ int64_t* daliShapeAt(daliPipelineHandle* pipe_handle, int n) {
   return daliShapeAtTypedHelper(pipe_handle, n, -1);
 }
 
+template <typename T>
+static dali_data_type_t daliTypeAtHelper(dali::DeviceWorkspace* ws, int n) {
+  const auto &out_tensor_list = ws->Output<T>(n);
+  auto type_id = out_tensor_list.type().id();
+  return static_cast<dali_data_type_t>(static_cast<int>(type_id));
+}
+
+dali_data_type_t daliTypeAtSample(daliPipelineHandle* pipe_handle, int n) {
+  return daliTypeAt(pipe_handle, n);
```

Comment:
What is the value in having `daliTypeAt`?

Reply:
To be able to access the type. And for the API to be complete

Reply:
But it is only passing through arguments to daliTypeAtSample. So I don't see much value in having two functions.

Reply:
Damn, forgot to remove it from here.